<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Descriptives</class>
 <widget class="QWidget" name="Descriptives">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>976</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fred</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_16">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_10">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="ExpanderButton" name="pushButton_3">
        <property name="text">
         <string>Statistics</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="pageStatistics" native="true">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="3" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Distribution</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_9">
              <property name="text">
               <string>Skewness</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_10">
              <property name="text">
               <string>Kurtosis</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QWidget" name="widget_5" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2" colspan="2">
          <widget class="QGroupBox" name="statistics__central_tendency">
           <property name="title">
            <string>Central Tendency</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="BoundCheckBox" name="statistics_centralTendency_mean">
              <property name="text">
               <string>Mean</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="BoundCheckBox" name="statistics_centralTendency_median">
              <property name="text">
               <string>Median</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="BoundCheckBox" name="statistics_centralTendency_mode">
              <property name="text">
               <string>Mode</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="BoundCheckBox" name="statistics_centralTendency_sum">
              <property name="text">
               <string>Sum</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="statisticsDispersion">
           <property name="title">
            <string>Dispersion</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QCheckBox" name="dispersionStdDeviation">
              <property name="text">
               <string>Std. deviation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="statisticsDispersionMinimum">
              <property name="text">
               <string>Minimum</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="dispersionVariance">
              <property name="text">
               <string>Variance</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="statisticsDispersionMaximum">
              <property name="text">
               <string>Maximum</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="dispersionRange">
              <property name="text">
               <string>Range</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="dispersionStdErrorMean">
              <property name="text">
               <string>S. E. Mean</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="statisticsPercentileValues">
           <property name="title">
            <string>Percentile Values</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
            <item row="0" column="0" colspan="2">
             <widget class="BoundCheckBox" name="statisticsPercentileValuesQuantiles">
              <property name="text">
               <string>Quantiles</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>equal groups</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="statisticsPercentileValuesEqualGroupsNumberOfGroups">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maxLength">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="BoundCheckBox" name="statisticsPercentileValuesEqualGroups">
              <property name="text">
               <string>Cut points for :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="BoundCheckBox" name="statisticsPercentileValuesPercentiles">
              <property name="text">
               <string>Percentiles :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="statisticsPercentileValuesPercentilesPercentiles">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="statisticsValuesAreGroupMidpoints">
              <property name="text">
               <string>Values are group midpoints</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QGridLayout" name="gridLayout_13">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="ExpanderButton" name="pushButton_4">
        <property name="text">
         <string>Charts</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="pageCharts" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Chart Type</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0" colspan="2">
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>None</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>Bar Charts</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>Pie Charts</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QRadioButton" name="radioButton_4">
              <property name="text">
               <string>Histograms</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="checkBox_17">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Show normal curve on histogram</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QWidget" name="widget_13" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Chart Values</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_5">
              <property name="text">
               <string>Frequencies</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_6">
              <property name="text">
               <string>Percentages</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="ExpanderButton" name="pushButton_5">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pageFormat" native="true">
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="2" column="2">
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Maximum number of categories</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QCheckBox" name="checkBox_19">
           <property name="text">
            <string>Suppress tables with many categories</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="widget_11" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Order by</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_7">
              <property name="text">
               <string>Ascending values</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_8">
              <property name="text">
               <string>Descending values</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButton_9">
              <property name="text">
               <string>Ascending counts</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="radioButton_10">
              <property name="text">
               <string>Descending counts</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Multiple Variables</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_11">
              <property name="text">
               <string>Compare variables</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_12">
              <property name="text">
               <string>Organise output by variables</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QWidget" name="widget_12" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>573</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="2">
       <widget class="BoundListView" name="main_fields">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="BoundCheckBox" name="main_displayFrequencyTables">
        <property name="text">
         <string>Display Frequency Tables</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="AvailableFieldsListView" name="listAvailableFields">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="AssignButton" name="buttonAssign_main_fields">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QWidget" name="widget_6" native="true">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpanderButton</class>
   <extends>QPushButton</extends>
   <header>widgets/expanderbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>assignClicked()</slot>
  <slot>unassignClicked()</slot>
  <slot>okClicked()</slot>
 </slots>
</ui>
